vochain/scrutinizer/scrutinizer.go

```diff
@@ -56,52 +63,103 @@ func NewScrutinizer(dbPath string, state *vochain.State) (*Scrutinizer, error) {
 		return nil, err
 	}
 	s.VochainState.AddEventListener(s)
+	go s.AfterSyncBootstrap()
 	return s, nil
 }
 
+func (s *Scrutinizer) AfterSyncBootstrap() {
+	// Some grace time to avoid false positive on IsSynchronizing()
```

Comment:
During the first seconds/milliseconds of the Vochain startup, Tendermint might report that the chain is not synchronizing since it still does not have any peer and do not know the actual size of the blockchain. If AfterSyncBootStrap is executed on this specific moment, the Wait loop would pass.

Reply:
Gotcha, maybe summarize that here?